tests/integration_tests.rs

```diff
@@ -894,7 +894,7 @@ async fn txn_scan_reverse() -> Result<()> {
     ];
 
     // pessimistic
-    let option = TransactionOptions::new_pessimistic().drop_check(tikv_client::CheckLevel::Warn);
```

Comment:
Why changes this?

Reply:
> Why changes this?

This case is unstable. Via debug, I found that `new_pessimistic` use `no_backoff` for `lock_backoff`, which is the reason to cause it.  `new_optimistic` will do retry when failure. It seems the transaction option is not the test point in this case, so I prefer to use the more stable one.

Reply:
Got it. Can you change the comment above and briefly describe the reason in it?

Reply:
Fixed. Thanks.